apps/desktop/src/renderer/styles/tables.css

```
@@ -0,0 +1,173 @@
+ *  C. Sortable preview tables
+ */
+
+/* ===================================================================
+   A. Editor WYSIWYG Tables
+   =================================================================== */
+
+.cm-table-widget {
+  overflow-x: auto;
+  margin: var(--space-3) 0;
+  border-radius: var(--radius-md);
+  overflow: hidden;           /* clip child table to border-radius */
```

Stylelint (17.7.0)

[error] Overridden property "overflow-x" by shorthand "overflow" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)